docs/content/components/form/form/form.mdx

```diff
@@ -0,0 +1,109 @@
+
+### Props
+
+<PropsTable
+  props={[
+    {
+      property: 'validationErrors',
+      type: 'Record<string, string | string[]>',
+      description:
+        'Validation errors for the form, typically returned by a server. This should be set to an object mapping from input names to errors.',
+      default: '-',
+    },
+    {
+      property: 'action',
+      type: `string | FormHTMLAttributes<HTMLFormElement>['action']`,
+      description:
+        'Endpoint where to send the form-data when the form is submitted.',
+      default: '-',
+    },
+    {
+      property: 'method',
+      type: `'get' | 'post' | 'dialog'`,
+      description: 'The HTTP method to submit the form with.',
+      default: '-',
+    },
+    {
+      property: 'onSubmit',
+      type: `(event: FormEvent<HTMLFormElement>) => void`,
+      description: 'Triggered when a user submits the form.',
+      default: '-',
+    },
+    {
+      property: 'onReset',
+      type: `(event: FormEvent<HTMLFormElement>) => void`,
+      description: 'Triggered when a user resets the form.',
+      default: '-',
+    },
+    {
+      property: 'onInvalid',
+      type: `(event: FormEvent<HTMLFormElement>) => void`,
+      description:
+        'Triggered for each invalid field when a user submits the form.',
+      default: '-',
+    },
+    {
+      property: '...',
+      type: '',
+      description: 'Yes you can use all regular attributes of `form`!',
+      default: '',
+    },
+  ]}
+/>
+
+## Examples
+
+### Setup
+
+This is a simple setup how to use a `<Form>`.
+
+<ComponentDemo file="./form-base.demo.tsx" />
+
+### Handling submission
+
+The `onSubmit` event is useful for custom form actions, such as calling a REST API, instead of relying on the native form submission. It triggeres when a user submits the form using the Enter key or clicks the submit button. The `onReset` event is triggered when a user presses a reset button (`[type=reset]`).
+
+<ComponentDemo file="./form-submission.demo.tsx" />
+
+### Server Errors
+
+The `<Form>` component handles passed errors, typically received from a server after form submission. To display validation errors, set the `validationErrors` prop as an object mapping each field's name prop to a string or array of strings representing errors. These errors appear to the user as soon as the `validationErrors` prop is set and are cleared when the user modifies the corresponding field's value.
+
+<ComponentDemo file="./form-validation-errors.demo.tsx" />
+
+For more information about form validation, see the [Validation](/concepts/validation) guide.
+
+### Focus Management
+
+As you can see in the previous server errors example, when a user submits a form with validation errors, the first invalid field is automatically focused. This behavior can be customized using `e.preventDefault` during the `onInvalid` event and manage the focus manually.
+
+<ComponentDemo file="./form-focus-management.demo.tsx" />
+
+<p />
```

Comment:
what is this? :o

Reply:
If have to add this otherwhise the message and the component demo don't have any spacing in between :D